archetypes/data-playground.md

```diff
@@ -1,23 +1,23 @@
 ---
 title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
 slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
-subtitle: "add subtitle here" 
+subtitle: "Add Data Type Here" 
```

Comment:
I believe that I added @bbengfort suggestions. I also tried to make the page a little more descriptive after it was mentioned during our last meeting. The order has been amended to match the Figma design to hopefully make things easier as changes are made and for reviewers.